src/components/policy/policy_external/include/policy/policy_table/enums.h

```diff
@@ -35,31 +35,19 @@
 
 #include <string>
 
+#include "generated_MOBILE_API_policy_types.h"
+#include "generated_HMI_API_policy_types.h"
+
 namespace rpc {
 namespace policy_table_interface_base {
 
-enum Priority {
-  P_EMERGENCY,
-  P_NAVIGATION,
-  P_VOICECOM,
-  P_COMMUNICATION,
-  P_NORMAL,
-  P_NONE,
-};
-
-bool IsValidEnum(Priority val);
-const char* EnumToJsonString(Priority val);
-bool EnumFromJsonString(const std::string& literal, Priority* result);
-
-enum HmiLevel {
-  HL_BACKGROUND,
-  HL_FULL,
-  HL_LIMITED,
-  HL_NONE,
+enum Input {
```

Comment:
@LuxoftAKutsan Is this enum used somewhere?

Reply:
@AKalinich-Luxoft yes in cache manager.